packages/pluggableWidgets/line-chart-web/rollup.config.js

```diff
@@ -66,7 +68,16 @@ export default args => {
                 }
             ]
         }),
-        production ? terser() : null
+        production ? terser() : null,
+        command([
+            () => {
+                const workerFilePath = join(
```

Comment:
Is this required by ace editor?

Reply:
Yes, it was throwing errors about a missing file. This was fixed in the original charts: https://github.com/mendix/charts/pull/138, but that fix doesn't work in monorepo cuz it uses webpack instead of rollup. The quickest solution that I came up with was this.